tests/foreman/ui/test_host.py

```diff
@@ -2517,6 +2517,67 @@ def _cleanup():
             ), 'Could not remove installed packages in a finalizer!'
 
 
+@pytest.mark.parametrize(
+    'module_repos_collection_with_setup',
+    [
+        {
+            'distro': 'rhel8',
+            'YumRepository': {'url': settings.repos.yum_3.url},
+        }
+    ],
+    ids=['yum3'],
+    indirect=True,
+)
+@pytest.mark.no_containers
+@pytest.mark.tier2
+@pytest.mark.rhel_ver_list([settings.content_host.default_rhel_version])
+@pytest.mark.rhel_ver_match('8')
+def test_all_hosts_manage_errata(
+    session,
+    module_target_sat,
+    module_org,
+    mod_content_hosts,
+    module_repos_collection_with_setup,
+    new_host_ui,
+):
+    """Apply an errata on multiple hosts through bulk errata wizard in All Hosts page.
+
+    :id: 41bca5f8-bf2d-4b16-974c-5abda3c09ee2
+
+    :expectedresults: Errata can be bulk applied to hosts through the All Hosts page.
+
+    :CaseComponent: Hosts-Content
+
+    :Team: Phoenix-content
```

Comment:
```suggestion
    :Team: Phoenix-subscriptions
```
Hosts-Content belongs to the subs team.

Reply:
This will never make sense to me......it's in the name.....